### PR TITLE
Pop window

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears,
You should implement popups for both mobile and desktop screen sizes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -387,10 +179,10 @@ <h3>Skills</h3>
           </ul>
         </div>
       </section>
-    </section>
+    </div>
 
     <footer>
-      <section class="form-section">
+      <section class="form-section" id="contact">
```
Comment (andersonlebon) [inline index.html:185]: ### [OPTIONAL]
- [ ]  I would kindly suggest you increase the width of your contact form(inputs) in the mobile version.
correct this design below 👇  👇 
your design
image

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+          </div>
+            <div class="modal-btns">
+              <a href="${project.gitLink}" class="btn github-link">
+                See live
+                <span class="iconify iconbtn" data-icon="bytesize:github"></span>                                                                      
+              </a>
+              <a href="${project.srcLink}" class="btn source-link">
+                See source                
+                <span class="iconify iconbtnlink" data-icon="cil:external-link"></span>                            
+              </a>
```
Comment (andersonlebon) [inline js/main.js:177]: - [ ] I would recommend also kindly sticking to the design of these two buttons(height && width only).

The template | your design 
--------------|----------
image | image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hello @ismailakinkunmi 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your popup is responsive
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ismailakinkunmi** [comment]
Thanks for the review, Those See buttons bug are clear to me but I can't
figure out anything at the tonic, canopy either desktop or mobile version,
i will suggest if you can take me through at zoom video for it, looking
forward to your response.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @andersonlebon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**